### ISSUE TITLE
spinner issue with ie9

### ISSUE THREAD

**Issue by smart987** [urish/angular-spinner#48]
Hi Urish, Thanks for the nice spinner. The example plunker given on github showing the following error on ie9.

[object Error]{description: "Unspecified...", message: "Unspecified...", name: "Error", number: -2147467259}

 In my angular js application if I comment out,  <span us-spinner="{radius:30, width:8, length: 16}" spinner-key="spinner-1"></span>,    my application working fine on ie9. But if I uncomment this line the application is not at all running and is displaying the above error. 
How can I fix it? Please see the error image for reference.

![spinnererror](https://cloud.githubusercontent.com/assets/11309381/7082195/6419883a-df6d-11e4-866f-1452d25dc75d.png)

**Comment by urish** [comment]
Can you for please share the plunkr link?

**Comment by smart987** [comment]
Urish, It is the same link that you have given in this github repo. http://plnkr.co/edit/BGLUYcylbIVJRz6ztbhf?p=preview but I have opened it in ie9.

**Comment by urish** [comment]
It seems like the error is related to plunkr and not the code itself. When I go to http://run.plnkr.co/LtWvW2Azlo6TMd99/, there are no errors in the console. Can you please confirm that is also the case for you?

**Comment by smart987** [comment]
Yes Urish, but it is also correct that my application is failing on IE9 when I put spinner directive in my jsp page, the soon I comment the spinner directive line in jsp page, my application is working fine on IE9. I am trying to provide you error replication code for your reference, soon I have it ready I will  update here. Thank you very much.

**Comment by smart987** [comment]
Sorry for the late, urish. The sample plunker you have given on Github is working fine even on IE9. But what I am trying to do is to start and stop the spinner in the controller using $scope.startSpin() and $scope.stopSpin() may be that is causing some issue. Please find the plunker http://plnkr.co/edit/00lrsMO8dMpBSots9zlI?p=preview. Here spinner is not even appearing.

**Comment by Attrash-Islam** [comment]
Hi @smart987 ,
Thanks for reporting.

Can you please reproduce the plunker again? It's not exist anymore
